src/main/java/eu/xenit/gradle/docker/tasks/internal/WorkaroundDockerdConsecutiveCopyBugAction.java

```diff
@@ -0,0 +1,153 @@
+package eu.xenit.gradle.docker.tasks.internal;
+
+import com.bmuschko.gradle.docker.tasks.image.Dockerfile;
+import com.bmuschko.gradle.docker.tasks.image.Dockerfile.CopyFile;
+import com.bmuschko.gradle.docker.tasks.image.Dockerfile.CopyFileInstruction;
+import com.bmuschko.gradle.docker.tasks.image.Dockerfile.Instruction;
+import java.util.ArrayList;
+import java.util.List;
+import javax.annotation.Nullable;
+import org.gradle.api.Action;
+import org.gradle.api.NonNullApi;
+import org.gradle.api.Task;
+import org.gradle.api.logging.Logger;
+import org.gradle.api.logging.Logging;
+
+@NonNullApi
+public class WorkaroundDockerdConsecutiveCopyBugAction implements Action<Task> {
+
+    private static final Logger LOGGER = Logging.getLogger(WorkaroundDockerdConsecutiveCopyBugAction.class);
+    public static final String FEATURE_FLAG = "eu.xenit.docker.flags.workaround-dockerd-consecutive-copy-bug";
+    private static final int MAX_CONSECUTIVE_COPIES = 6;
```

Comment:
Should we make this configurable as well? Is it somehow possible this changes on different docker versions / hosts?

Reply:
Good point. An extra property does not really cost much anyways and saves us from having to release a new version when we need to tweak it.